### DISCUSSION TITLE
Shlink Does Not Start with Service

### DISCUSSION THREAD

**Discussion by TheAFKGamer10** [shlinkio/shlink#1889] [Help wanted] [answered]
### Shlink version

v3.6.4

### PHP version

8.2.10

### How do you serve Shlink

Self-hosted RoadRunner

### Database engine

MySQL

### Database version

8.0.34 - MySQL Community Server - GPL

### Summary

When starting Shlink with Roadrunner normaly with the `./bin/rr serve -c config/roadrunner/.rr.yml` command it works correctly. But if I try to start it with the rr.service file it does not seam to start. 

Here is my current rr.service file as stated in the [RR Docs](https://roadrunner.dev/docs/app-server-systemd/2.x/en).
```service
[Unit]
Description=High-performance PHP application server

[Service]
Type=simple
ExecStart=/var/www/shlink/bin/rr serve -c config/roadrunner/.rr.yml
Restart=always
RestartSec=30

[Install]
WantedBy=default.target 
```

**Comment by acelaya** [comment, marked as answer]
What do you mean exactly by `it does not seam to start`? 
Have you checked the service status with `systemctl status rr`?
Have you tried passing an absolute path to the `.rr.yml` config file?
The docs you linked to are outdated. Take a look at the right one, which has a bit more information https://roadrunner.dev/docs/app-server-systemd/current/en
